SU-SWS/stanford_bean_types
# Single centered button view mode SOE-3292

# READY FOR REVIEW


# Summary
- Removing the soe-default view mode from the postcard block.
- Adding the single_centered_button view mode.

# Needed By (Date)
- Sprint end

# Criticality
- Client is eagerly awaiting this for the release.

# Steps to Test
- `drush fr stanford_bean_types --force -y`
- Go to `admin/structure/ds/view_modes`
- Make sure you see `Single Centered Button` and do NOT see `SoE Default`

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)

## Related PRs

## More Information

## Folks to notify
@boznik 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
